stable/prometheus-adapter/templates/custom-metrics-configmap.yaml

```diff
@@ -13,6 +13,24 @@ data:
 {{- if or .Values.rules.default .Values.rules.custom }}
     rules:
 {{- if .Values.rules.default }}
+    - seriesQuery: 'node_memory_Active_bytes'
```

Comment:
we use the upstream prometheus-adapter chart in [kommander-applications](https://github.com/mesosphere/kommander-applications/blob/main/services/prometheus-adapter/4.2.0/helmrelease/prometheus-adapter.yaml#L15) (not this chart)

Instead of updating this chart, we will want to add this config here: https://github.com/mesosphere/kommander-applications/blob/main/services/prometheus-adapter/4.2.0/defaults/cm.yaml#L21

Reply:
ok, thanks. I'll close this one